README.md

```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
 }
 ```
 
+---
+
+## SNC DNS en tant que service
+
+Avec nos remerciements à nos amis de [18f](https://github.com/18F) pour l'[inspiration](https://github.com/18F/dns), ce dépôt est une collection de scripts Terraform pour la configuration des DNS pour les domaines gérés par SNC.
+
+~~Actuellement, il se limite à la gestion des sous-domaines de *.cds-snc.ca.  À l'avenir, nous étendrons cette liste pour inclure le domaine cdssandbox.xyz.~~
+Pour le moment, nous gérons les domaines:
+
+- _*.cds-snc.ca_
+- _*cdssandbox.xyz_
+- _*.alpha.canada.ca_.
+
+### Pourquoi en aurais-je besoin?
+
+Vous voulez une jolie URL comme `notification.alpha.canada.ca` pour votre nouveau projet ?  C'est comme ça qu'il faut faire.
+
+### Faire des requêtes DNS
+
+Pour demander un nouveau sous-domaine, créez une entrée de terraforme `aws_route53_record` et soumettez un [enjeu (issue)](https://github.com/cds-snc/dns/issues/new) ~~pull request~~.
+Un membre de l'équipe de la plate-forme examinera la demande et l'approuvera ou reviendra avec des questions supplémentaires.
+
+Nous cherchons à faciliter les choses, mais si quelqu'un veut nous aider, veuillez contacter john.obrien@cds-snc.ca
+
```

Comment:
```suggestion
```

Our dear @obrien-j is leaving CDS soon so removing this sentence

Reply:
Yes, just learned! Sad to see him go!